### PR TITLE
Blog app - creating a data model

### PR DESCRIPTION
**Pull request by edi-sipka**
- Created a new app. 
- Initialized  project with Git.
- Project has Postgres database set up.
- Built  schema.
- Created and run the necessary migration files.
- Table and column names matched the ERD diagram.
- Foreign keys are included.
- All columns that are foreign keys have a corresponding index.

### CODE REVIEW

File: db/migrate/20221122131303_add_users_ref_to_posts.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddUsersRefToPosts < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :user, null:false, foreign_key:true
+  end
```
Comment (youmari) [inline db/migrate/20221122131303_add_users_ref_to_posts.rb:4]: - [ ]  as you can see in the diagram it should be `author_id`not `user_id ` so please make sure to fix that  by trying to change the name of the column you can do it like in the below example: 
```ruby
 add_reference :(table name) :author, foreign_key: { to_table: 'users' }
 ```

File: db/migrate/20221122131354_add_users_ref_to_comments.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class AddUsersRefToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :user, null:false, foreign_key:true
```
Comment (youmari) [inline db/migrate/20221122131354_add_users_ref_to_comments.rb:3]: - [ ]  as you can see in the diagram it should be `author_id`not `user_id ` so please make sure to fix that  by trying to change the name of the column you can do it like in the below example: 
```ruby
 add_reference :(table name) :author, foreign_key: { to_table: 'users' }
 ```

File: db/migrate/20221122131421_add_users_ref_to_likes.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class AddUsersRefToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, null:false, foreign_key:true
```
Comment (youmari) [inline db/migrate/20221122131421_add_users_ref_to_likes.rb:3]: - [ ]  as you can see in the diagram it should be `author_id`not `user_id ` so please make sure to fix that  by trying to change the name of the column you can do it like in the below example: 
```ruby
 add_reference :(table name) :author, foreign_key: { to_table: 'users' }
 ```

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @edi-sipka , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Great commit messages
✔️ PRO PR
✔️ No linter error 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShadyShawkat — APPROVED** [review]
## Hi @edi-sipka, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- All linters are passing ✔️ 
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 
- Your readme looks professional ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**